src/proxy.c

```diff
@@ -32,6 +32,24 @@ r_obj* vec_proxy_2(r_obj* x, bool recurse) {
 
   case VCTRS_TYPE_s3: {
     r_obj* out = vec_proxy_invoke(x, info.proxy_method, recurse);
```

Comment:
I think `out` needs to be protected now, because you might call `r_clone()`, which allocates?